dashboard/client/src/api/functionsApi.js

```diff
@@ -1,11 +1,11 @@
 import axios from 'axios';
 import moment from 'moment';
+import {createNew} from "typescript";
```

Comment:
How come we're importing typescript into React?

Reply:
Think this was an IDE auto-import. removed